TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/PopulatePersonSearchAttributesJob.cs

```diff
@@ -0,0 +1,90 @@
+using Hangfire;
+using Microsoft.EntityFrameworkCore;
+using TeachingRecordSystem.Core.DataStore.Postgres;
+using TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+namespace TeachingRecordSystem.Core.Jobs;
+
+[AutomaticRetry(Attempts = 0)]
+public class PopulatePersonSearchAttributesJob(TrsDbContext dbContext)
+{
+    public async Task Execute(Guid personId)
+    {
+        var person = await dbContext.Persons.AsNoTracking().SingleAsync(p => p.PersonId == personId);
```

Comment:
Can we call the SQL function that the trigger uses here instead of duplicating logic?